docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -65,9 +65,58 @@ boosted tree regression model which combines multiple weak models into a
 composite one. We use decision trees to learn to predict the probability that 
 a data point belongs to a certain class.
 
+
+[discrete]
+[[dfa-classification-interpret]]
+==== Interpreting {classification} results
+
+The following sections help you understand and interpret the results of a 
+{classanalysis}.
+
+[discrete]
+[[dfa-classification-class-probability]]
+===== `class_probability`
+
+The value of `class_probability` shows how likely it is that a given datapoint 
+belongs to a certain class. It is a value between 0 and 1. The higher the 
+number, the higher the probability that the data point belongs to the named 
+class. For further details, see the
```

Comment:
I think we need to qualify this link a bit more, since the example focuses on viewing the results in Kibana, which I don't think shows this class_probability value by default.  For example:
```suggestion
class. This information is stored in the `top_classes` array for each document in your destination index. See the
```